__tests__/components/App.test.js

```diff
@@ -20,63 +24,70 @@ describe('<App />', () =>{
     expect(wrapper.find('div#app').length).toEqual(1)
   })
 
-  it('renders <Footer />', () => {
-    expect(wrapper.find(Footer).length).toBe(1)
+  it('renders the LoginPanel component', () => {
+    expect(wrapper.find(LoginPanel).length).toEqual(1)
   })
 
+  it('renders the Footer component', () => {
+    expect(wrapper.find(Footer).length).toBe(1)
+  })
 })
 
 describe("#routes", () => {
+  // pattern cribbed/adapted from https://stackoverflow.com/a/54807560
+  const makeStoreFake = state => ({
+      default: () => {
+      },
+      subscribe: () => {
+      },
+      dispatch: () => {
+      },
+      getState: () => ({ ...state })
+    });
 
   describe('public routes', () => {
-    const renderRoutes = path =>
-      mount(
-        <MemoryRouter initialEntries={[path]}>
-          <RootContainer />
-        </MemoryRouter>
-      )
+    const unauthenticatedStoreFake = makeStoreFake({
+        authenticate: {
+          authenticationState: {
+            currentSession: null
+          }
+        }
+      })
 
-    const renderEditorRoute = path =>
+    const renderRoutes = path =>
       mount(
-        <MemoryRouter initialEntries={[path]}>
-          <RootContainer>
-            <App>
-              <Editor location={{state: { resourceTemplateId: 'resourceTemplate:bf2:Monograph:Instance' }}} />
-            </App>
-          </RootContainer>
-        </MemoryRouter>
+        <Provider store={unauthenticatedStoreFake}>
+          <MemoryRouter initialEntries={[path]}>
+            <App />
+          </MemoryRouter>
+        </Provider>
       )
 
-    it('root renders HomePage component', async () => {
+    it('root renders HomePage component', () => {
```

Comment:
For the tests in this block, I might expect to see `test()` instead of `it()`, but since they're aliases... this is merely stylistic. (When I see `it()`, I want the test description to begin with a verb, but it makes sense for these tests to start with a noun to contextualize them.)

Reply:
will keep the `it` style since that seems consistent with other tests, and will verbify and regularize the titles accordingly.